extensions/pkg/controller/backupbucket/actuator.go

```diff
@@ -12,10 +12,18 @@ import (
 	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
 )
 
-// Actuator acts upon BackupBucket resources.
+// Actuator acts upon [extensionsv1alpha1.BackupBucket] resources.
 type Actuator interface {
-	// Reconcile reconciles the BackupBucket.
+	// Reconcile reconciles the [extensionsv1alpha1.BackupBucket] resource.
+	//
+	// Implementations should take care for creating or updating the
+	// BackupBucket in order to bring it to its desired state.
 	Reconcile(context.Context, logr.Logger, *extensionsv1alpha1.BackupBucket) error
-	// Delete deletes the BackupBucket.
+
+	// Delete deletes the BackupBucket associated with the
+	// [extensionsv1alpha1.BackupBucket] resource.
+	//
+	// Implementations must wait until the BackupBucket is gracefully
+	// deleted.
```

Comment:
```suggestion
	// Delete deletes the extensionsv1alpha1.BackupBucket resource.
	//
	// Implementations should ensure that the associated backup bucket is gracefully deleted.
```

To make it more consistent to the doc string of the `Reconcile` function.

Reply:
>  Delete deletes the extensionsv1alpha1.BackupBucket resource.

Not sure that the suggested change is correct. 

The `Delete` method doesn't really delete an `extensionsv1alpha1.BackupBucket` resource, it is supposed to delete the respective bucket associated with this resource, but the actual resource is not deleted by it.

Thoughts?